examples/Mass Groups & Collaborations Update/README.md

```diff
@@ -0,0 +1,132 @@
+# Mass Groups & Collaborations Update #
+
+## Overview ##
+The Mass Groups and Collaborations Update powershell script will use the Box CLI to create or update groups, add users to them and finally to create collaborations for groups to the folders.
```

Comment:
General minor comment for Readme: Please do syntax check. (E.g. powershell -> PowerShell)